pkgs/top-level/all-packages.nix

```diff
@@ -23739,6 +23739,8 @@ in
 
   thepeg = callPackage ../development/libraries/physics/thepeg { };
 
+  xflr5 = libsForQt5.callPackage ../applications/science/physics/xflr5 { };
```

Comment:
@esclear there is a section `### PHYSICS` with other "interactive" apps.
Look for sherpa or xfitter for example.

Reply:
Okay, moved it.